tests/fs_diff/test_fs_diff.py

```diff
@@ -14,13 +15,7 @@ def test_fs_diff(hosts):
 
     fsdiff = os.path.realpath(f"{os.path.dirname(__file__)}/../../scripts/xcpng-fs-diff.py")
 
-    process = subprocess.Popen(
-        [fsdiff, "--reference-host", f"{hosts[0]}", "--test-host", f"{hosts[1]}", "--json-output"],
-        stdout=subprocess.PIPE, stderr=subprocess.PIPE
-    )
-    stdout, _ = process.communicate()
-
-    if process.returncode != 0:
-        print(stdout.decode())
-
-    assert process.returncode == 0
+    res = local_cmd([fsdiff, "--reference-host", f"{hosts[0]}",
+                             "--test-host", f"{hosts[1]}",
+                             "--json-output"], simple_output=False)
+    assert res.returncode == 0
```

Comment:
Doesn't this change the behavior, when we were only showing `stdout` when a non-expected diff occurs?

Reply:
Absolutely !
Fixed